SU-HSDO/suhumsci
# SHS-44 Implement new Color pairing for Traditional Theme

# READY FOR REVIEW

## Summary
Added the new `FireFinch` theme to the traditional theme

## Need Review By (Date)
This was in the backlog

## Urgency
This was in the backlog

## Steps to Test
1. Go [here](http://sparkbox-sandbox.suhumsci.loc/) (please be on the shs-44 branch)
2. On the homepage, open up the devtools and inspect this 
![image](https://user-images.githubusercontent.com/72449433/171881249-47491c03-cb4c-42e7-8ef1-4c28fded5a2a.png)
3. Check what the `--palette-primary-color` is 
![image](https://user-images.githubusercontent.com/72449433/171881466-f61531be-4a13-4cd9-afac-2167ad77333f.png)
4. These colors should match `firefinch` you chan check that by looking in this file: `docroot/themes/humsci/humsci_basic/src/scss/settings/_variables.traditional-pairings.scss`
5. If the colors do not align, navigate [here](http://sparkbox-sandbox.suhumsci.loc/admin/appearance/settings/humsci_traditional): and change the default theme to `firefinch` 
![image](https://user-images.githubusercontent.com/72449433/171893373-6bd6e48f-990d-4b95-9183-d6c157f36bbe.png)
6. Then change the [site theme](http://sparkbox-sandbox.suhumsci.loc/admin/appearance): 
![image](https://user-images.githubusercontent.com/72449433/171894104-5142bbfb-c9c2-4617-aae1-1b12d320f667.png)


## Extra Notes: 

In this file `docroot/themes/humsci/humsci_basic/tests/sass-specs/tools/_functions.color-pairings.scss` : , I changed this line 

```
@include assert-equal($test, $expect, 'The Warbler color pairing primary color value either does not match or does not exist.');
```

Previous it said "The Bluejay color pairing......", I believe that it was meant to say "The Warbler color pairing". 

This was a change for `Warbler` and not `firefinch`. So please let me know if this change is ok, if not I can revert it.

Also `firefinch` does not have values for. I don't think this is a problem, but there will be warnings about this in the command line.

```
"tertiary-highlight-darken-10": 
    "tertiary-reversed": 
    "tertiary-reversed-darken-10": 
    "tertiary-darken-20": 
    "spotlight": 

```

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
